svg.js.d.ts

```diff
@@ -754,10 +754,10 @@ declare namespace svgjs {
     // selector.js
     interface Library {
         get(id: string): Element;
-        select(query: string, parent: HTMLElement): Set;
+        select(query: string): Set;
     }
     interface Parent {
-        select(query: string): Set;
+        select(query: string, parent: HTMLElement): Set;

```

Comment:
On the other hand `SVG.Parent.select()` does NOT have a second parameter. The parent is already defined because you called the method on a parent (like in `group.select(query)`. Are you sure you didnt mix up something here?

Reply:
Like in above, this change is not necessary, so will bring it back and try to push changes in the evening.